internal/customerrors/error.go

```diff
@@ -45,5 +45,5 @@ func getLogFileMessage() string {
 	}
 
 	return fmt.Sprintf("For more information, check the log file at %s\n"+
-		"nancy version: %s\n", logFile, buildversion.BuildVersion)
+		"ahab version: %s\n", logFile, buildversion.BuildVersion)
```

Comment:
THANK YOU FOR FIXING WHAT IS LIKELY JEFFRY BEING VERY COPY PASTA MANE (don't even have to git blame this to know my fingers and command+c +v were moving faster than my brain)